src/Build/BackEnd/Components/Logging/LoggingService.cs

```diff
@@ -1651,8 +1651,11 @@ private void UpdateMinimumMessageImportance(ILogger logger)
                 // The null logger has no effect on minimum verbosity.
                 Execution.BuildManager.NullLogger => null,
 
-                // If the logger is not on our whitelist, there are no importance guarantees. Fall back to "any importance".
-                _ => MessageImportance.Low
+                // The live logger consumes only high priority messages.
+                _ => innerLogger.GetType().FullName == "Microsoft.Build.Logging.LiveLogger.LiveLogger"
```

Comment:
minor: consider `typeof(LiveLogger).FullName`, that way if namespaces change the code will remain valid.

Reply:
This is unfortunately not possible with the current layering. The new logger is in `MSBuild.dll` and this backend code resides in `Microsoft.Build.dll` which is a dependent assembly of `MSBuild.dll`. We are aware of the limitation and if more issues like this arise, we may go ahead and move the logger to the other assembly.